### DISCUSSION TITLE
Calling unknown method: craft\elements\Asset::thumbUrl()

### DISCUSSION THREAD

**Discussion by ankitupadhyay360** [solspace/craft-freeform#1291] [Questions & Answers] [answered]
After form submission when  I want to see submitted data form craft admin this error occurs why ??

Calling unknown method: craft\elements\Asset::thumbUrl()

 in C:\xampp\htdocs\sandyou\vendor\solspace\craft-freeform\packages\plugin\src\templates\submissions\fields\file.twigat line 12
3456789101112131415161718192021        <label>{{ field.label }}</label>
    </div>
    <div class="file-preview">
        {% set assets = submission.getAssets(field.handle) %}
 
        {% if assets %}
            {% for asset in assets|filter(asset => asset is not null) %}
                {% set thumbSizes = [30, 60, 100, 200] %}
                {% set srcset = "" %}
                {%- for size in thumbSizes -%}{% set srcset = srcset ~ ", " ~ asset.thumbUrl(size) ~ " " ~ size ~ "w" %}{%- endfor -%}
                {% set srcset = srcset|trim(", ") %}
 
                <div class="element small removable hasthumb" style="padding-right: 20px;">
                    <input type="hidden" name="{{ field.handle }}[]"
                           value="{{ asset.id }}">
                    <a href="javascript:;" onclick="removeAsset(this);" class="icon delete"></a>
                    <div class="elementthumb">
                        <img sizes="30px" srcset="{{ srcset }}" alt="">
                    </div>

**Comment by kjmartens** [comment, marked as answer]
Sorry about this issue @ankitupadhyay360.

This is now resolved in Freeform 5.3.0+ 🙂
